tutorial/part-1-creating-a-simple-view/creating-your-first-table.md

```diff
@@ -0,0 +1,141 @@
+# Creating your first page
+
+Now that your environment – an “app” – is set up, you’re set to start doing work. 
+
+Each application that you build in Appsmith, comprises of one or more pages. A page in Appsmith is where the three building blocks of your app - Widgets, APIs, and DB Queries come together to create a view. 
+
+To view all the products at Oakry, we will create a page having a table that lists these products. A page was already created when you created a new page. Let's use the rename that page **Page1** to **ProductListPage**:
+
+1. Click on **⋮** next to **Page1**
+2. Choose **Edit Name**
+3. Type **ProductListPage**
+
+![](https://lh5.googleusercontent.com/0rI5_YOs-6SUuU5AKpCKQCio2Q6jqbyc1YZuoRr9zZ8Vfa3QsEq45ol0z6qFiD1bA5Fw_gN6UZgDLVmXRl9P5gSaarNSaXK8MK1AmqPaqYZ80i1v0wF58bMMNIqDil5FoI4tjvjv)
+
+## Adding your first widget
+
+The **ProductListPage** is blank. Let’s add a table-widget to it.
+
+1. Navigate to **Product List → Widgets**
+2. Click on **+**
+3. Choose the **table widget** 
+4. Drag and drop the table widget to the page on the right
+
+This will create a new table. 
+
+![](https://lh4.googleusercontent.com/p6VRCgNSNPxyq1IdSgVbU7oHE8fkTDmayGM-YPIuOBKHCzEhE2qYYaTyDQ6XyCG7xmQ6CoNlUCBTO6iat52sZqs8Ig8GzOLFpDF2_3GEXgGcSgwMmOuba5Pekv1ZY3roaOgr5EI0)
+
+Let’s take a look at what you now see on screen:
+
+* You can see that the table is populated with some default static data.
+* You see a modal, titled **Table1**, open up on the right of the table. This modal is where you configure the widget’s properties. 
```

Comment:
I chose modal because of the explanation [here](https://ux.stackexchange.com/questions/90336/whats-the-difference-between-a-modal-popup-popover-and-lightbox). Changing to floating window; and finally referring as property-pane